fab/fabfile.py

```diff
@@ -391,21 +357,6 @@ def install_npm_packages():
 @task
 @roles(ROLES_ALL_SRC)
 @parallel
-def upgrade_packages():
-    """
-    Bring all the installed packages up to date.
-    This is a bad idea in RedHat as it can lead to an
-    OS Upgrade (e.g RHEL 5.1 to RHEL 6).
-    Should be avoided.  Run install packages instead.
-    """
-    _require_target()
-    if what_os() == 'ubuntu':
-        sudo("apt-get update", shell=False, user='root')
-        sudo("apt-get upgrade -y", shell=False, user='root')
-    else:
-        return
-
-
 @task
```

Comment:
as in you don't think `what_os` should be a task? i'm down with that. also down with thinking about removing that entire function